test/esinstall/dep-list-simple/__snapshots__

```diff
@@ -7,7 +7,7 @@ Array [
 ]
 `;
 
-exports[`dep-list-simple matches the snapshot: cli output 1`] = `""`;
+exports[`dep-list-simple matches the snapshot: cli output 1`] = `"@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to \`true\`, as the next major version will default this option to \`true\`."`;
```

Comment:
_Makes PR to bubble up errors_

_Errors bubble up in snapshots_

_MFW_

![](https://en.meming.world/images/en/thumb/2/2c/Surprised_Pikachu_HD.jpg/300px-Surprised_Pikachu_HD.jpg)